### PR TITLE
feat: Created Left navbar, search results list, and search result components

### LINKED ISSUE #37: Create Search Page - Search Results Components - Desktop View
# Requirements

- A list that shows healthcare professionals
- use dummy data for milestone 1
- match styling from prototype

## Workflow - Using the results list

- User can see results list and it renders properly in desktop mode

Edit: updated after developer meeting to reflect new requirements

### PR DESCRIPTION
**Pull request by ermish**
Resolves #37

This PR includes new components for the Left Navbar, Search Results List, and the Search Result. 

# Tasks 
- [x] Created basic components
- [x]  Clean up styling
- [x] Fix page layout to support new left navbar

# Testing instructions
- Open up the app and view the new left nabber with results listed

### CODE REVIEW

**Comment by ermish** [comment]
ok @theyokohamalife @ann-kilzer! 
Styling is added and I think it's good enough for review now. 
It's not perfect but good enough to merge in.

File: components/SearchResult.vue [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                height="25" />
+            <span class="flex-1 w-4 font-bold pl-2"> {{ name }} </span>
+        </div>
+        <div class="result-details flex flex-row my-2">
+            <span class="pr-2"> {{ specialty }}</span>
+            <span class="self-center">·</span>
+            <span class="px-2"> {{ facilityName }}</span>
+        </div>
+        <div class="result-tags flex flex-row flex-wrap w-64 mb-2">
+            <div :key="index" v-for="(spokenLanguage, index) in spokenLanguages" class="pl-2 pr-2 py-[1px] mr-2 border border-solid border-orange/40 rounded-2xl shadow text-sm">
```
Comment (ann-kilzer) [inline components/SearchResult.vue:14]: nit: line too long

**Review by ann-kilzer — APPROVED** [review]
I think it's really coming along. Nice to see some structure to the page. 🌟 

1. I notice it doesn't look very clear on mobile. Can you make a follow-up PR to ensure responsive design?

https___deploy-preview-131--findadoc_netlify_app_and_List_of_ISO_639-1_codes_-_Wikipedia

2. Consider adding a few lightweight Cypress tests at some point just to prevent regressions

File: tailwind.config.js [hunk 2]
```diff
@@ -15,10 +15,13 @@ module.exports = {
         screens: {
             md: { min: '800px' }
         },
-        extend: {},
         colors: {
             transparent: 'transparent',
             current: 'currentColor',
+            currentColor: '#ED6C5A',
+            orange: '#ED6C5A',
```
[the remark this answers is not in the document]
Comment (ermish) [reply tailwind.config.js:22]: makes sense to me! I just copied the colors straight out of Kei's prototype for now. 👍

File: components/SearchResultsList.vue [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    spokenLanguages: ['japanese', 'english']
+}, {
+    name: 'test person 2',
+    specialty: 'Psychiatrist',
+    facilityName: 'Rest Assured',
+    spokenLanguages: ['english']
+}, {
+    name: 'test person 3',
+    specialty: 'Oncologist',
+    facilityName: 'Bring me back to life',
```
[the remark this answers is not in the document]
Comment (ermish) [reply components/SearchResultsList.vue:36]: ⚡

File: components/SearchResult.vue [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+                {{ spokenLanguage }}
+            </div>
+        </div>
+    </div>
+</template>
+
+<script setup>
```
[the remark this answers is not in the document]
Comment (ermish) [reply components/SearchResult.vue:21]: yup! you beat me to it. I'm going to add in a follow up issue. I started with it and ran into some vue type issues, so skipped for this PR.

**Comment by ermish** [comment]
> I notice it doesn't look very clear on mobile. Can you make a follow-up PR to ensure responsive design?
@ann-kilzer mochiron! We've got #126.  For right now, I'm knocking out desktop features until we have mobile mockups. 😸

File: components/SearchResult.vue (hunk 1 above)
Comment (ermish) [reply components/SearchResult.vue]: updated

File: components/SearchResult.vue (hunk 4 above)
[the remark this answers is not in the document]
Comment (ermish) [reply components/SearchResult.vue:22]: created #137

File: tailwind.config.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (ann-kilzer) [reply tailwind.config.js:22]: Ah I see. I sent him a note about this too in slack. In my experience black or white text and icons work best against various colors (color in the art school sense 🌈 🎨  where black and white are values).

**Pull request merged**